README.md

```diff
@@ -77,6 +77,7 @@ A collection of algorithms for _Distributed Systems_ course (winter semesters 20
 
 #### Synchronized maximal independent set
 1. Luby algorithm
+2. Métivier et al. algorithm (by Métivier, Robson, Saheb-Djahromi, and Zemmari)
```

Comment:
Just "Métivier-Robson-Saheb-Djahromi-Zemmari algorithm" is fine

Reply:
I put the full name in `README.md` and started referring to the algorithm as "Métivier C" everywhere else